src/test/java/com/github/dockerjava/core/util/CertificateUtilsTest.java

```diff
@@ -0,0 +1,28 @@
+package com.github.dockerjava.core.util;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+public class CertificateUtilsTest {
```

Comment:
That assumes that the resource files as found in the classloader are accessible in the filesystem with a base of the project, I assume. In case the test would run from a JAR it should fail. In that respect using MyTestClass.class.getName().replace(".", File.separator) plus some additions seems a bit better to me as it has no dependency to the classloader. Building the path correctly is however still not nice as no working dir is known.
A save approach would be to create a temp file and place the required files in there as needed. I didn't find anything like that in the existing coda and thus did not do that. Also the cleanup is not so nice.